Dockerfile

```diff
@@ -1,10 +1,7 @@
 FROM node:10
-RUN mkdir -p /usr/src/lobby
-WORKDIR /usr/src/lobby
-COPY package.json /usr/src/lobby/
-COPY package-lock.json /usr/src/lobby/
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
```

Comment:
Changed the `WORKDIR` to something more general since the resulting image would be used by both the `node` and `lobby` images.